fission/src/systems/physics/PhysicsSystem.ts

```diff
@@ -777,122 +768,6 @@ class PhysicsSystem extends WorldSystem {
         }
     }
 
-    // TODO: Ball socket joints should try to be reduced to the shoulder joint equivalent for Jolt (SwingTwistConstraint)
```

Comment:
Are ball socket joints implemented? I'm kinda hesitant to remove this because I don't know what it does.

Reply:
This is my reminder to myself to finish going through all the stuff that Hunter was working on but never committed.